app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -409,7 +409,7 @@ private static String getFileContentType(File f) {
      */
     public static boolean isSupportedMultimediaFile(String filename) {
         for (String ext : SUPPORTED_FILE_EXTS) {
-            if (filename.endsWith(ext)) {
+            if (filename.toLowerCase().endsWith(ext)) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/943db269bf6832e319807f4f0ed4715c5c38ea7a/app/src/org/commcare/utils/FormUploadUtil.java#L412"></span>Implicitly using the default locale is a common source of bugs: Use <code>toLowerCase(Locale)</code> instead. For strings meant to be internal use <code>Locale.ROOT</code>, otherwise <code>Locale.getDefault()</code>.</td>
    </tr>
  </tbody>
</table>

Reply:
Wow, I don't know about this^. Should I use `toLowerCase(Locale.English)`?

Reply:
yeah we should use `Locale.English` to be safe